Formula/simple-tiles.rb

```diff
@@ -1,10 +1,9 @@
 class SimpleTiles < Formula
   desc "Image generation library for spatial data"
   homepage "https://github.com/propublica/simple-tiles"
-  url "https://github.com/propublica/simple-tiles/archive/v0.6.1.tar.gz"
-  sha256 "2391b2f727855de28adfea9fc95d8c7cbaca63c5b86c7286990d8cbbcd640d6f"
+  url "https://github.com/e-n-f/simple-tiles/archive/refs/tags/v0.7.0.tar.gz"
```

Comment:
If you are not going to maintain it, we can simply deprecate this formula, no need to cut a new release for gdal

Reply:
yeah, make sense.